backend/shared/shared/oidc/views/eauth_views.py

```diff
@@ -79,6 +79,11 @@ def get(self, request):
             ).replace("http://", "https://"),
             "user": user_id,
         }
+
+        lang = request.session.get("lang")
```

Comment:
Wondering if it would be useful to also set the `django_language` cookie? (that's the default for `LANGUAGE_COOKIE_NAME` 
My understanding is that Django Rest Framework would then use the cookie value for localized validation errors? (in Helsinki benefit, the frontend may display the validation errors to the end user so we need to localize them, but it depends on your use case)

Reply:
This sounds like a good idea! I'll make some changes.